lib/grizzly/zwave/commands/node_provisioning_set.ex

```diff
@@ -43,7 +43,7 @@ defmodule Grizzly.ZWave.Commands.NodeProvisioningSet do
   def encode_params(command) do
     seq_number = Command.param!(command, :seq_number)
     meta_extensions = Command.param!(command, :meta_extensions)
-    dsk = Command.param!(command, :dsk)
+    dsk = dsk_bin_from_command!(command)
     dsk_byte_size = byte_size(dsk.raw)
```

Comment:
I believe this line needs to be adjusted too? The new function is returning a binary but this call is expecting the struct

Reply:
🤦‍♂️  Fixed and added tests.